opbeat/base.py

```diff
@@ -162,6 +162,14 @@ def __init__(self, organization_id=None, app_id=None, secret_token=None,
             msg = 'Missing configuration for client. Please see documentation.'
             self.logger.info(msg)
 
+        if is_enabled is None and os.environ.get('OPBEAT_IS_ENABLED'):
+            msg = "Configuring opbeat from environment variable 'OPBEAT_IS_ENABLED'"
+            self.logger.info(msg)
+            is_enabled = os.environ['OPBEAT_IS_ENABLED'] == 'True'
```

Comment:
I don't think so, any string evaluated to a boolean will result to True, even 'False'.
Don't you think this can be a strange behavior for the user point of view?